allvers/2006-11-30-GPL-2.0-from-fsf.org.txt.notes

```diff
@@ -0,0 +1,4 @@
+Date introduced/changed: 20061130102107
```

Comment:
Incorrect. This one was introduced much earlier, namely between May 16, 2006 and May 28, 2006 (this is 6a on my list)

Reply:
Fixed in 03d5734971df5dfe2a9fca319ad79a82de458eff